models/classes/runner/QtiRunnerService.php

```diff
@@ -76,6 +76,7 @@
 use qtism\data\AssessmentItemRef;
 use qtism\runtime\tests\SessionManager;
 use oat\libCat\result\ResultVariable;
+use League\Flysystem\FileNotFoundException;
```

Comment:
This is not the exception caught before

Reply:
it's actually the exception caught afterwards, but I believe neither can be triggered